optimize-data-indexing/README.md

```diff
@@ -0,0 +1,63 @@
+---
+page_type: sample
+languages:
+  - csharp
+name: Optimize Indexing with the Push API
+description: Learn how to efficiently index data using Azure Cognitive Search's Push API and an exponential backoff retry strategy. This tutorial and sample code are in C#.
+products:
+  - azure
+  - azure-cognitive-search
+urlFragment: optimize-data-indexing
+---
+
+# Optimize Indexing with the Push API
+
+![Flask sample MIT license badge](https://img.shields.io/badge/license-MIT-green.svg)
+
+This .NET Core console app builds off of the code used in the [Quickstart](https://docs.microsoft.com/en-us/azure/search/search-get-started-dotnet) and uses the [Azure Cognitive Search .NET SDK](https://docs.microsoft.com/dotnet/api/?term=microsoft.azure.search) to create an index and efficiently load it with documents.
+
+The app shows how to:
+
+- Test different batch sizes to understand the optimal batch size for indexing your data
+- Efficiently index data by:
+  - [Pushing](https://docs.microsoft.com/en-us/azure/search/search-what-is-data-import#pushing-data-to-an-index) data to the index in batches
+  - Sending multiple batches concurrently
+  - Implementing an [exponential backoff retry strategy](https://docs.microsoft.com/en-us/dotnet/architecture/microservices/implement-resilient-applications/implement-retries-exponential-backoff)
+
+The index is modeled on a subset of the Hotels dataset, reduced for readability and comprehension. Index definition and documents are included in the code.
+
+> **Note:**  Depending on your network transfer speeds, you may need to run this sample from within your Azure Environment to get the most out of Azure Cognitive Search's indexing speed. 
```

Comment:
Azure environment is essentially just saying to run it from within an azure datacenter. Upload speeds are significantly faster on a VM vs my local pc just because my home network can't upload data as fast as search can ingest it. I'll plan to merge your paragraph with mine